pkg/runtime/ast/ast_test.go

```diff
@@ -245,6 +245,7 @@ func TestParse_ShowStatement(t *testing.T) {
 		{"show table status from foo where name='bar'", (*ShowTableStatus)(nil), "SHOW TABLE STATUS FROM `foo` WHERE `name` = 'bar'"},
 		{"show nodes from arana", (*ShowNodes)(nil), "SHOW NODES FROM `arana`"},
 		{"show users from arana", (*ShowUsers)(nil), "SHOW USERS FROM `arana`"},
+		{"show sharding table from employees", (*ShowShardingTable)(nil), "employees"},
```

Comment:
why the expected is `employees`?

Reply:
Because in line 555 of the file “ pkg\runtime\ast\show.go” ：
```go
func (s *ShowShardingTable) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
	val, ok := s.baseShow.filter.(string)
	if !ok {
		return errors.New("show sharding table database type error !")
	}
	sb.WriteString(val)
	return nil
}
```
I  directly used the value of the `val` variable instead of `SHOW SHARDING TABLE FROM val` to facilitate obtaining the database name directly in Plan by:
`dbName = sb.String() `.
Do I need to make any modifications?